### PR TITLE
TW-1758: Create upload file on web use dio

### PR DESCRIPTION
**Pull request by nqhhdev**
### Ticket

- #1758

### CODE REVIEW

**Review by Te-Z — DISMISSED** [review]

File: lib/data/network/media/media_api.dart [hunk 1]
```diff
@@ -31,6 +35,33 @@ class MediaAPI {
...
+    return UploadFileResponse.fromJson(response);
+  }
+
+  Future<UploadFileResponse> uploadFileWeb({
+    required MatrixFile file,
+    CancelToken? cancelToken,
+    ProgressCallback? onReceiveProgress,
+  }) async {
+    final dioHeaders = _client.getHeaders();
+    dioHeaders[HttpHeaders.contentLengthHeader] = file.bytes!.length;
```
Comment (hieutbui) [inline lib/data/network/media/media_api.dart:53]: Can it lead to an error?

Comment (Te-Z) [reply lib/data/network/media/media_api.dart:53]: @nqhhdev wdyt ?

Comment (nqhhdev) [reply lib/data/network/media/media_api.dart]: Updated

**Comment by nqhhdev** [comment]
Rebased, ready to merge

**Review by hieutbui — APPROVED** [review]

**Review by sherlockvn — APPROVED** [review]

**Review by Te-Z — APPROVED** [review]

**Pull request merged**